share/revng/test/tests/scripting/scripting.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
```

Comment:
Two things:
* Since this is the only file in the directory, move it to the parent
* `chmod +x` this file since it's an executable script, drop the `python ...` in the `scripting.yml`